bluemira/builders/EUDEMO/ivc/ivc.py

```diff
@@ -282,9 +294,27 @@ def _make_flux_surface_keep_out_zone(self, psi_n: float) -> BluemiraWire:
         Make a "keep-out zone" from an equilibrium's flux surface.
         """
         flux_surface_zone = self.equilibrium.get_flux_surface(psi_n)
-        flux_surface_zone = make_polygon(flux_surface_zone.xyz, closed=True)
+        # Chop the flux surface to only take the upper half
+        indices = np.where(flux_surface_zone.z >= self.o_points[0][1])[0]
```

Comment:
Not sure that you need the `np.where` here, you can index directly with the boolean array

```suggestion
        indices = flux_surface_zone.z >= self.o_points[0][1]
```